dbms/src/Storages/Transaction/LearnerRead.cpp

```diff
@@ -457,6 +459,7 @@ void validateQueryInfo(
         if (auto iter = regions_snapshot.find(region_query_info.region_id); //
             iter == regions_snapshot.end() || iter->second != region)
         {
+            // If snapshot is applied during learner read, we should abort.
```

Comment:
what do you mean "abort" here? abort the tiflash process or throw an exception to abort the read request?

Reply:
I will make this clear